### ISSUE TITLE
Incorrect TS type for authorizationStatus in PushNotificationPermissions

### ISSUE THREAD

**Issue by cmrigney** [react-native-push-notification-ios/push-notification-ios#196]
The type returned by `PushNotificationIOS.requestPermissions(...)` is `PushNotificationPermissions` which has a property `authorizationStatus`. The real type of the `authorizationStatus` property is a number but the [TS type has it as an interface](https://github.com/react-native-push-notification-ios/push-notification-ios/blob/29733b3c545bf83dc6c82356adbbd98187d8b45f/index.d.ts#L11).

Currently, I have to do this to bypass the TypeScript error:
```
const permission = await PushNotificationIOS.requestPermissions(...)
const authorized = ((permission.authorizationStatus as any) as number) === PushNotificationIOS.AuthorizationStatus.UNAuthorizationStatusAuthorized
```

**Comment by Naturalclar** [comment]
@cmrigney thanks for raising the issue!
The fix has been applied and published in v1.6.1 :tada:

**Comment by cmrigney** [comment]
@Naturalclar That was fast, thanks!
